ATM.py

```diff
@@ -0,0 +1,112 @@
+"""The script simulates ATM.
+
+The script simulates ATM that preforms the following:
+balance check, withdraw cash, deposit cash and
+password change.
+"""
+import os
+import sys
+
+
+BALANCE = 1
+PASSWORD = 0
+ID_INDEX = 0
+PATH_TO_FILE = 1
+INDEX_START_OF_PASSWORD_AND_BALANCE = 1
+
+
+def file_to_dic(filename):
+    """Insert the ATM  data from the file into the dictionary
+
+    Args:
+        filename (str): file name ending with '.txt'
+
+    Returns:
+        dictionary: key=id and value=[password, balance]
+    """
+    atm = {}
+    with open(filename, 'r') as file:
+        for line in file:
+            details = line.strip().split(' ')
+            atm[details[ID_INDEX]] = details[INDEX_START_OF_PASSWORD_AND_BALANCE:]
+
+    return atm
+
+
+def operation_of_atm(id, atm_dic):
+    """Performs the desired ATM operation for the inserted id.
+
+    Args:
+        id (str): the id for which you want the ATM to work
+        atm_dic (dict): dictionary containing all ATM information
+
+    Returns:
+        dictionary: dictionary with the updated values
+            after performing the desired action
+    """
+    if id in atm_dic:
```

Comment:
generally very good job in this function.
Try to go with convention that says if you got a function with more then 15~20 lines of code, then it usually means that you do more then one functionality in this function and you can split it to several smaller functions